diff --git a/includes/errors.inc b/includes/errors.inc
index 7393148..bc85f65 100644
--- a/includes/errors.inc
+++ b/includes/errors.inc
@@ -138,7 +138,7 @@ function _drupal_render_exception_safe($exception) {
 /**
  * Determines whether an error should be displayed.
  *
- * When in maintenance mode or when error_level is ERROR_REPORTING_DISPLAY_ALL,
+ * When error_level is ERROR_REPORTING_DISPLAY_ALL,
  * all errors should be displayed. For ERROR_REPORTING_DISPLAY_SOME, $error
  * will be examined to determine if it should be displayed.
  *
@@ -150,12 +150,11 @@ function _drupal_render_exception_safe($exception) {
  */
 function error_displayable($error = NULL) {
   $error_level = variable_get('error_level', ERROR_REPORTING_DISPLAY_ALL);
-  $updating = (defined('MAINTENANCE_MODE') && MAINTENANCE_MODE == 'update');
   $all_errors_displayed = ($error_level == ERROR_REPORTING_DISPLAY_ALL);
   $error_needs_display = ($error_level == ERROR_REPORTING_DISPLAY_SOME &&
     isset($error) && $error['%type'] != 'Notice' && $error['%type'] != 'Strict warning');
 
-  return ($updating || $all_errors_displayed || $error_needs_display);
+  return ($all_errors_displayed || $error_needs_display);
 }
 
 /**
